.gitignore

```diff
@@ -3,3 +3,4 @@ phpunit.xml
 composer.lock
 humbuglog.txt
 coverage
+.idea
```

Comment:
This change shouldn't be here: please use a [global `.gitignore`](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)